scripts/vmf/train-vmf-enfr.sh

```diff
@@ -3,6 +3,7 @@ python main.py \
     --device cuda \
     --dropout 0.0 \
     --enc_bidirectional \
+#    --enc_num_layers 2\
```

Comment:
This will comment out the rest of the lines in the command, if I am not mistaken, because `\` is a line continuation and `#` is a line comment.

Reply:
Ok so I'll move it to the end of the command.